wp-content/themes/vdc_2.4/footer.php

```diff
@@ -11,31 +11,31 @@
 
 	elseif($parent == 6){
 		$parent_page = "org"; }
-		
+
 	$page_id = $post->ID;
 	$bg_img = get_field('background_image', 'options');
 	$logo = get_field('logo_footer', 'options');
 	$mobile_logo = get_field('mobile_logo', 'options');
 ?>
 			</main>
-				
+
```

Comment:
That's a good point. We may need to define some rules to configure our IDEs settings consistently.